src/page-modules/contact/components/input/input.module.css

```diff
@@ -0,0 +1,41 @@
+@value typo-body__secondary from "@atb/theme/typography.module.css";
+
+.container {
+  composes: typo-body__secondary;
+
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 100%;
+  color: var(--static-background-background_0-text);
+  background-color: var(--static-background-background_0-background);
+  gap: var(--spacings-small);
+  margin-bottom: var(--spacings-medium);
+}
+
+.radio_or_checkbox {
+  flex-direction: row-reverse;
+  justify-content: start;
+  gap: var(--spacings-small);
+  margin-bottom: var(--spacings-medium);
+}
+
+.input {
+  padding: var(--spacings-small);
+  color: var(--static-background-background_0-text);
+  outline: 0;
+  background: none;
+  border-radius: 0.75rem;
+  max-width: 400px;
```

Comment:
I think it's better to let the parent component decide the max-width of an input